web_dialog_size/static/src/js/web_dialog_size.js

```diff
@@ -1,37 +1,41 @@
-openerp.web_dialog_size= function (instance) {
-
-    instance.web.Dialog =  instance.web.Dialog.extend({
-
-        init_dialog: function () {
-            var self = this;
-            this._super();
-            self.$dialog_box.find('.dialog_button_restore').addClass('dialog_button_hide');
-            if (this.dialog_options.size !== 'large'){
-                self.$dialog_box.find('.dialog_button_extend').addClass('dialog_button_hide');
-            }
-            else{
-                self.$dialog_box.find('.dialog_button_extend').on('click', self._extending);
-                self.$dialog_box.find('.dialog_button_restore').on('click', self._restore);
+odoo.define('web_dialog_size.web_dialog_size', function (require) {
+'use strict';
+
+var Model = require('web.DataModel');
+var Dialog = require('web.Dialog');
+
+Dialog.include({
+
+    init: function (parent, options) {
+        var self = this;
+        this._super.apply(this, arguments);
+        self.$modal.find('.dialog_button_extend').on('click', self.proxy('_extending'));
+        self.$modal.find('.dialog_button_restore').on('click', self.proxy('_restore'));
+
+        new Model('ir.config_parameter').query(['key', 'value']).
+        filter([['key', '=', 'web_dialog_size.default_maximize']]).all().then(function(default_maximize) {
```

Comment:
Why not use the get_param method if ir.config_parameter as in v8? This should be more efficient than querying then filtering.